servers/src/main/java/tachyon/master/next/filesystem/FileSystemMaster.java

```diff
@@ -256,11 +257,13 @@ public long getFileId(TachyonURI path) throws InvalidPathException {
     }
   }
 
-  public FileInfo getFileInfo(long fileId) throws FileDoesNotExistException {
+  public FileInfo getFileInfo(long fileId) throws FileDoesNotExistException, InvalidPathException {
     // TODO: metrics
     synchronized (mInodeTree) {
       Inode inode = mInodeTree.getInodeById(fileId);
-      return inode.generateClientFileInfo(mInodeTree.getPath(inode).toString());
+      FileInfo fileInfo = inode.generateClientFileInfo(mInodeTree.getPath(inode).toString());
+      fileInfo.inMemoryPercentage = getInMemoryPercentage(mInodeTree.getPath(inode));
```

Comment:
This should be done to the getFileInfoList() method as well. (Are there any other methods returning FileInfo?)

Also, add simple comment saying this has to be filled here, and not in the InodeFile class.

Also, if only this method uses the getInMemoryPercentage() method, maybe it should take in a fileId instead of path (and make it private), so it doesn't have to traverse the inode tree again for the path.